### ISSUE TITLE
Import metadata/text for missing textbooks

### ISSUE THREAD

**Issue by lmullen** [lmullen/legal-modernism#94]

**Comment by lmullen** [comment]
Data should be in two tables.

BOOK METADATA:
bibliographicid (ignore)
year
title
vols
subjects (can be an array)
psmid (make something up)
author (can also add) (e.g., `Abbott, Benjamin Vaughan; Barringer, Victor Clay`)

PAGE OCR DATA, one row per page
pageid (format: page 1 = `00010`
psmid (make it up, this identifies the volume)
ocrtext

**Comment by lmullen** [comment]
For psmids use something like this:

momlextra00001

**Comment by kfunk074** [comment]
Metadata here: 
[side_corpus.csv](https://github.com/user-attachments/files/16897543/side_corpus.csv)

Page OCR data is stored here: https://drive.google.com/drive/folders/1vzENEoxKK74cAI_m5qpVLLBFl9moKW3P?usp=sharing

Each folder is labeled with its psmid, and each page is a separate text file in the folder.

**Comment by lmullen** [comment]
Yes, I agree that we should do this before we do the citation detection.

I'm pretty sure I am going to add these to the `moml.*` and just treat them as if they were in MOML in some way. It will drastically simplify everything downstream.

**Comment by lmullen** [comment]
For adding these, it it is important that we use the same fields in the tables in the `moml` database schema that we actually use in the Go programs. Results of running `ag --go "moml." --nocolor`:

```
go/citations/store_db.go
26:     output.moml_citations (id, moml_treatise, moml_page, raw, volume, reporter_abbr, page, created_at)

go/sources/treatise.go
5:// TreatisePage represents a page from a treatise in MOML.

go/sources/store_pgx.go
36:     SELECT psmid, pageid, ocrtext FROM moml.page_ocrtext
57:     query := `SELECT psmid, pageid FROM moml.page_ocrtext;`
112:    FROM moml.page_ocrtext po
113:    JOIN moml.book_info bi ON po.psmid = bi.psmid
116:    FROM moml.book_subject bs
```

Those are the relevant places to check which fields are actually used.
